app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+    def index
+    end
+end
```

Comment:
Looks like `app/views/home/index.html.erb` is an empty file, maybe we can just remove this controller and view? and just route the root to `students#index`